src/main/java/racingcar/domain/Position.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.domain;
+
+import java.util.Objects;
+
+public class Position {
+    public static final Position ZERO = new Position(0);
+
+    private static final int ONE = 1;
```

Comment:
```suggestion
    private static final Position ONE = new Position(1);
```

해당 값 또한 객체로 바꿔서 처리한다면 더욱 객체지향적인 코드가 되겠네요!!

Reply:
좋습니당 👍

Reply:
Position이 add와 같은 메서드를 제공한다면 int가 아닌 Position으로 처리가 가능할 것 같은데 어떻게 생각하시나요!
물론 지금 작성하신 것 처럼 원시값을 사용해도 충분히 훌륭한 코드네요 ㅎㅎ

Reply:
ㅎㅎ position이 2씩 바뀌도록 변경되더라도 대응할 수 있겠네요! 더 나은 방식이라고 생각합니다